geozero/src/lib.rs

```diff
@@ -28,6 +28,9 @@
 //! | WKB       | [Wkb](wkb::Wkb), [Ewkb](wkb::Ewkb), [GpkgWkb](wkb::GpkgWkb)          | XYZM       | -                                                                    | [ToWkb]             | [WkbWriter](wkb::WkbWriter)             |
 //! | WKT       | [wkt::WktStr], [wkt::WktString]                                      | XYZM       | [wkt::WktReader], [wkt::WktStr], [wkt::WktString]                    | [ToWkt]             | [WktWriter](wkt::WktWriter)             |
 
+#[macro_use]
+extern crate log;
```

Comment:
I usually include the macros like this for log crate, so that I can `info!` instead of `log::info!`, but happy to revert it if it's controversial.

Reply:
I usually `use log::info`, but I'm also fine with this.

Reply:
I've removed this in favor of `log::warn!`, I don't have strong feelings about it either and now that csv is optional it was producing an "unsused" error. 🙃